tests/languages/php/class-name_feature.test

```diff
@@ -0,0 +1,34 @@
+Foo::bar();
+(Foo $variable)
+): Foo
+): ?Foo
+
+----------------------------------------------------
+
+[
+	["class-name", ["Foo"]],
+	["punctuation", ":"],
+	["punctuation", ":"],

```

Comment:
These two should be one token. Please overwrite the `punctuation` pattern with `/::|[{}[\];(),.:]/`. To do this, add the following lines after the `class-name` token:

```diff
 Prism.languages.php = Prism.languages.extend('clike', {
 	....
 	'class-name': {
 		...
-	}
+	},
+	'punctuation': /::|[{}[\];(),.:]/
 });
```

Reply:
double colon `::` is actually operator (scope / static access) -> moved to operators